vue.config.js

```diff
@@ -25,22 +25,15 @@ module.exports = {
       .plugin('favicons')
       .use(FaviconsWebpackPlugin, [{
         logo: path.resolve(__dirname, 'src/assets/favicon.svg'),
-        mode: 'webapp', // optional can be 'webapp' or 'light' - 'webapp' by default
-        devMode: 'webapp', // optional can be 'webapp' or 'light' - 'light' by default
         publicPath: '/',
         favicons: {
           start_url: '/',
-          appName: 'You Do Not Need A Cape To Be A Hero',
+          appName: 'Superhero.com',
           appDescription: 'You Do Not Need A Cape To Be A Hero',
           developerName: 'Aeternity Developers',
-          developerURL: 'https://github.com/kenodressel/aepp-boilerplate',
-          background: '#ff0d6a',
-          theme_color: '#ff0d6a',
-          icons: {
-            coast: false,
-            yandex: false,
-            windows: false,
-          },
```

Comment:
I can't find an explanation of why it is disabled in git blame. I think it is ok to use all features implemented in this package.